api/db/db_models.py

```
@@ -2002,6 +2002,27 @@ def alter_db_drop_index(migrator, table_name, index_name):
+        if "prefetch_size" not in columns:
+            return
+        if "rerank_candidates_count" in columns:
+            DB.execute_sql(
+                "UPDATE dialog SET rerank_candidates_count = prefetch_size"
+            )
+            DB.execute_sql("ALTER TABLE dialog DROP COLUMN prefetch_size")
+        else:
+            DB.execute_sql(
+                "ALTER TABLE dialog RENAME COLUMN prefetch_size TO rerank_candidates_count"
+            )
+    except Exception as ex:
```

Ruff (0.16.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)